core_backend/tests/api/test_data_api.py

```diff
@@ -161,9 +162,8 @@ def test_query_data_api_date_filter(
         client: TestClient,
         user1_data: pytest.FixtureRequest,
     ) -> None:
-
-        start_date = datetime.utcnow() - relativedelta(days=start_day)
-        end_date = datetime.utcnow() - relativedelta(days=end_day)
+        start_date = datetime.now(timezone.utc) - relativedelta(days=start_day)
```

Comment:
For some reason this test now fails for one of its cases... Not sure why. Any ideas @sidravi1?

Reply:
It is still failing? I see there is an error in the `test_query_data_api_other_user` test though. I'll fix that.

Reply:
yeah just one of these failing for me...